src/OpenLoco/src/Map/Track/Track.h

```diff
@@ -18,6 +18,16 @@ namespace OpenLoco::World::Track
     };
     static_assert(sizeof(TrackConnections) == 0x24);
 
+    namespace AdditionalTaDFlags
+    {
+        constexpr uint16_t basicTaDMask = 0b0000'0001'1111'1111;
+        constexpr uint16_t bridgeMask = 0b0000'1110'0000'0000;
```

Comment:
Should these be part of the same ~type~ namespace? I think we've avoided doing so in other places.

Reply:
weve done it like this in the imageid struct i think